messaging/src/main/java/org/axonframework/javax/common/jpa/EntityManagerProvider.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2010-2023. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.javax.common.jpa;
```

Comment:
I'd suggest changing the package order. Instead: `org.axonframework.common.jpa.javax` or `org.axonframework.common.javax-jpa`. Or maybe, in order to avoid issues the cause the rename from javax to jakarta: `org.axonframework.common.legacy-jpa`

Reply:
Moved to `legacy_jpa` packages.